### PR TITLE
Enabled HR (Croatian) translations

### PR DESCRIPTION
**Pull request by stefanb**
[Stanje prevoda](https://hosted.weblate.org/engage/sledilnik/hr/?utm_source=widget) - mostly by @milotype, very [active translator on Weblate](https://hosted.weblate.org/user/milotype/)  👏 

Preview available at https://pr-414.sledilnik.org/hr/stats

### CODE REVIEW

**Comment by milotype** [comment]
I have just made some minor corrections ...

File: src/i18n.js [hunk 1]
```diff
@@ -47,24 +49,45 @@ i18next.services.pluralResolver.addRule(
...
+    'travnja',
+    'svibnja',
+    'lipnja',
+    'srpnja',
+    'kolovoza',
+    'rujna',
+    'listopada',
+    'studenoga',
+    'prosinca',
+  ],
```
Comment (stefanb) [inline src/i18n.js:89]: @miaerbus, should that come from translations in `hr.json`?

**Comment by stefanb** [comment]
I have added  `*_hr.md` static pages to  https://github.com/sledilnik/website/tree/l10n-hr/src/content with short content consisting of:
- link to original page in slovenian
- link to page translated in english 
- link to page on github, so that it can be translated (it leads to master branch, where there are no _hr.md files until this PR is merged)

Example: https://pr-414.sledilnik.org/hr/links:
image

This way 
- user is not left in the dark with completely empty pages, 
- we don't have duplicated content and 
- maybe we get some translations :)

**Comment by milotype** [comment]
If you don't mind, I'd suggest to use the informal speech in Croatian – "ti" instead of "vi".
Also, IMO, imperative form is better than the conditional form. 

So the Croatian text would be:
"Nažalost, ova stranica još nije prevedena. Pogledaj izvorni tekst na slovenskom ili engleski prijevod."
and
"Pomogni nam prevesti ovu stranicu na GitHub-u."


Milo

**Comment by stefanb** [comment]
@milotype, the name of the source "[Our World In Data](https://ourworldindata.org/)" was reverted from "Naš svijet u brojkama" back to the original name in https://github.com/sledilnik/website/commit/1a82f9aeefa73bdbe9301f89d02e1aa761dbbc01

I think this should be ok, as googling  "Naš svijet u brojkama" revealed no connection to the original site.

**Comment by milotype** [comment]
I agree ... thank you.

**Pull request merged**